examples/azure-linux-tdx-ita-attestation-vm/README.md

```diff
@@ -0,0 +1,188 @@
+
+The TDX Intel Confidential Computing VM is hardened from the cloud virtualized environment by denying the hypervisor, other host management code and administrators access to the VM memory and state. The virtual machine is created on an Azure Standard_DC2es_v5 by default.
+
+
+Supported Intel Confidential Computing VMs with Intel TDX include:
+-DCesv5-series
+-DCedsv5-series
+-ECesv5-series
+-ECedsv5-series
+
+See root policies.md for full list of Intel Confidential VMs with TDX.
+
+By default this example will provision Azure "Standard_DC8es_v5" instance, feel free to change the size of the VM as needed.
+
+Azure VM Security Type will be set to Confidential amd Virtualized Trusted Platform Module (vTPM) enabled as requried with optional Secure Boot, OS disk encrypted at host.
+
+As you configure your application's environment, choose the configurations for your infrastructure that matches your application's requirements. 
+
+In this example, the virtual machine will be deplyed in a pre-existing Azure Virtual Network, with defualt subnet, and resource group. Make sure to the resoruce group is in the region where Intel Confidential Compute VMs with TDX is available. See variable.tf for the default cofigruation which you can change as needed.
+
+The tags Name, Owner and Duration are added to the virtual machine when it is created.
+
+Once the TDX Intel Confidential Computing VM with Intel Trust Authority is created, it will validate successfull configuation of Intel Trust Authority with your ITA Token with the "ITA-Verify.tok" file. You can view the ITA_Verify.tok in the trustauthority-client/tdx-cli folder to validate.
+
+NOTE: This exampel will take about 10min to complete as there is a 5min pause while runing the ansible playbook to allow Intel Trust Authority setup to complete before completing validattion. 
+
+After the buildout is complete, you will need to assiciate your Public IP to the newely created TDX Azure isntance and configure you Network Security Group for SSH etc as needed.
+
+## Usage
+
+** See examples folder for code ./examples/azure-linux-multi-disks/main.tf **
+
+variables.tf
+
+```hcl
+
+################################################################################################
+# Replace the default values with your own values - your trustauthority_api_key (ITA token)    #
+################################################################################################
+# Variable for trustauthority_api_key
+variable "trustauthority_api_key" {
+  description = "trustauthority_api_key"
+    type        = string
+    default     = "<YOUR ITA TOKEN HERE>="
```

Comment:
Comment out this line and make bring up more attention to the need to modify it.

Reply:
Done